pkgs/by-name/ta/tastytrade/package.nix

```diff
@@ -0,0 +1,116 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, makeBinaryWrapper
+, alsa-lib
+, atk
+, at-spi2-atk
+, cairo
+, cups
+, dbus
+, expat
+, freetype
+, glib
+, glibc
+, gtk3
+, libdrm
+, libGL
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, xdg-utils
+, xorg
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tastytrade";
+  version = "2.5.4";
+
+  src = fetchurl {
+    url = "https://download.tastytrade.com/desktop-2.x.x/${finalAttrs.version}/tastytrade-${finalAttrs.version}-1_amd64.deb";
+    hash = "sha256-j1qTMeaxHLkpjpYL6zf3Q7aT4aR94LRG7tW3ZUHkCp0=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeBinaryWrapper
+  ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    alsa-lib
+    atk
+    at-spi2-atk
+    cairo
+    cups
+    dbus
+    expat
+    freetype
+    glib
+    glibc
```

Comment:
```suggestion
```
I think glibc is provided by `stdenv` already.

Reply:
Whoops, nice catch. I didn't even notice!